### PR TITLE
Native plugins provider

### PR DESCRIPTION
**Pull request by lathamaniTech**
Created native plugins folder for all native plugins related functionality.

### CODE REVIEW

**Review by KarthickTechie — APPROVED** [review]
docs: added comments
added comments in deviceDetails service file

this annotation is wrong , replace docs :  -> refactor : . docs are only applicable for ReadMe.md

File: src/providers/NativeProviders/camera.ts
Comment (KarthickTechie) [inline src/providers/NativeProviders/camera.ts:1]: @line67 - image size 1024 shoud be read from GlobalAppConfigConstants.ts

Comment (KarthickTechie) [reply src/providers/NativeProviders/camera.ts:1]: line 87 , 87 - relative file directory path should be dynamic , remove static path

File: src/utility/AppConstants.ts
Comment (KarthickTechie) [inline src/utility/AppConstants.ts:1]: AppConstants should only consist Static Constants and should not have any Interfaces / types details

**Review by KarthickTechie — CHANGES_REQUESTED** [review]
resove code review comments

File: src/utility/AppConstants.ts
Comment (lathamaniTech) [reply src/utility/AppConstants.ts:1]: removed from AppConstants.

File: src/providers/NativeProviders/camera.ts
Comment (lathamaniTech) [reply src/providers/NativeProviders/camera.ts:1]: created GlobalAppDataConfig.ts file and read from this file. unused static path removed, if need will get dynamically.

**Pull request merged**